### PR TITLE
110.m

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

**Comment by d-w-moore** [comment]
Now that I look at it: this is contingent on this branch being merged into main branch of this plugin, see the inclusion of https://github.com/irods/irods_rule_engine_plugin_python/pull/111/commits/a26b0bd6e4c811f586d8b43ce55a9d00d608ca67 in order to make this build with irods_development_environment

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -19,8 +19,8 @@ def install_building_dependencies(externals_directory):
                       'irods-externals-avro1.9.0-0',
                       'irods-externals-boost1.67.0-0',
```
Comment (alanking) [inline irods_consortium_continuous_integration_build_hook.py:20]: I think we are on 1.68.0 in main

https://github.com/irods/irods/blob/0cd02fc73f721b8b5c86d20f5d52608c7675adaf/CMakeLists.txt#L24

**Review by alanking — COMMENTED** [review]
Looks good to me. You're right, we do need to cherry-pick that change to main, so I think it's fine as you have it here.

Please mark as ready for review if true.

File: irods_consortium_continuous_integration_build_hook.py (hunk 1 above)
Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py:20]: And by that of course I mean 1.78.0

Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py:20]: And... well, there are others that could be bumped too. Y'know, if you wanted. :)

Comment (d-w-moore) [reply irods_consortium_continuous_integration_build_hook.py:20]: Sounds like a plan.

Comment (trel) [reply irods_consortium_continuous_integration_build_hook.py:20]: we want the good versions, dan.

Comment (d-w-moore) [reply irods_consortium_continuous_integration_build_hook.py:20]: Should all externals dependencies be bumped to the maximum version available in core-dev?

Comment (trel) [reply irods_consortium_continuous_integration_build_hook.py:20]: i think so - yes?

Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py:20]: I think... it's more important to match whatever is in the CMakeLists.txt, but in general, main will be using whatever the latest is, I think. That's not always obvious by the package names tho

Comment (d-w-moore) [reply irods_consortium_continuous_integration_build_hook.py:20]: I was also going by the /usr/lib/irods/IRODSConfig.cmake for 4.3.0, but an ldd on the .so file generated by the python plugin really only seems to care about boost.  I think the rest of what's listed in the build hook is just copy-paste from other plugins.

Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py:20]: That is probably very true lol Should we trim it down to the essentials? Or maybe just for this change we can update the piece(s) we care about?

Comment (trel) [reply irods_consortium_continuous_integration_build_hook.py:20]: pare it down.   reduce future shenanigans.

Comment (d-w-moore) [reply irods_consortium_continuous_integration_build_hook.py]: Done.

**Review by alanking — APPROVED** [review]
Very nice. # em

**Comment by d-w-moore** [comment]
#'d

**Pull request merged**